src/sentry/static/sentry/app/components/group/externalIssueActions.tsx

```diff
@@ -37,87 +37,146 @@ class ExternalIssueActions extends AsyncComponent<Props, State> {
     this.state = {
       showModal: false,
       action: 'create',
-      selectedIntegration: this.props.integration,
-      issue: this.getIssue(),
+      selectedIntegration: null,
       ...this.getDefaultState(),
+      ...this.linkedIssuesFilter(),
     };
   }
 
   getEndpoints() {
     return [];
   }
 
-  getIssue() {
-    return this.props.integration && this.props.integration.externalIssues
-      ? this.props.integration.externalIssues[0]
-      : null;
+  linkedIssuesFilter() {
+    return this.props.configurations.reduce(
+      (acc, curr) => {
+        if (curr.externalIssues.length) {
+          acc.linked.push(curr);
+        } else {
+          acc.unlinked.push(curr);
+        }
+        return acc;
+      },
+      {linked: [] as GroupIntegration[], unlinked: [] as GroupIntegration[]}
+    );
   }
 
-  deleteIssue(issueId: string) {
-    const {group, integration} = this.props;
-    const endpoint = `/groups/${group.id}/integrations/${integration.id}/?externalIssue=${issueId}`;
+  deleteIssue(integration: GroupIntegration) {
+    const {group} = this.props;
+    const {externalIssues} = integration;
+    let issue = externalIssues[0];
```

Comment:
@NisanthanNanthakumar what happens if there are more than one external issues?

Reply:
@scefali we currently only account for one issue. https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/components/group/externalIssueActions.tsx#L50-L54
I don't think there will be a case where there is multiple external issues.
For example, we shouldn't have more than 1 jira ticket created for an issue for each jira configuration.

Reply:
@NisanthanNanthakumar fair enough. We should probably comment that though so it's clear.

Reply:
@NisanthanNanthakumar can you add this comment?